DSharpPlus/Entities/DiscordMessageBuilder.cs

```diff
@@ -45,9 +45,9 @@ public string Content
         /// <summary>
         /// Gets the Files to be sent in the Message.
         /// </summary>
-        public IReadOnlyDictionary<string, Stream> Files => this._files;
+        public IReadOnlyDictionary<string, Stream> Files => this._files.ToDictionary(x => x.Key, x => x.Value.Stream);
```

Comment:
You don't want to have this called every time a user calls this getter. You should instead initialize it either here or in the constructor.

Reply:
Yes i do, i want the list to only be pushed to _files which If the user calls Files it will then get its dictionary from _files.  I don't want to be managing both _files and Files.

Reply:
I will second Neuheit on this - creation of a dictionary per call seems rather wasteful.

Reply:
So we want to have 2 sets of dictionaries managed then?

Reply:
Yeah I think that's the only approach, unless anyone else has a solution.

Reply:
After discussing with @Neuheit in Discord, we changed it from being a Dictionary to a List of structs and making the List ReadOnly for the public API